develop/2017-04-29-fabiocorreacordeiro-Local-Transportation.py

```diff
@@ -0,0 +1,568 @@
+
+# coding: utf-8
+
+# # Exploratory Descriptive Analysis of Local Transportation
+# 
+# This is an Exploratory Descriptive Analysis of expenditure with local transportation. We already created three separated notebooks to explore the data of the subquotas 'Taxi, toll and parking', 'Automotive vehicle renting or charter' and 'Fuels and lubricants'. In this notebook, we will join the analysis of this subquotas.
+# 
+# The subquota 'Automotive vehicle renting or charter' changed in the time. It started like 'Automotive vehicle renting or watercraft charter' and in October/2013 was split in 'Watercraft renting or charter' and 'Automotive vehicle renting or charter'. 
+# 
+# The previous notebooks are in:
+# 
+# 'Automotive vehicle renting or charter': https://github.com/datasciencebr/serenata-de-amor/pull/206/commits/6afda4344c6565ec01536242353730b14f2ebaee
+# 
+# 'Taxi, toll and parking': https://github.com/datasciencebr/serenata-de-amor/blob/master/develop/2017-03-15-fabiocorreacordeiro-taxi-descriptive-analysis.ipynb
+# 
+# 'Fuels and lubricants': https://github.com/datasciencebr/serenata-de-amor/pull/203/commits/2635528e4faa9d94562e08972164ddba2173b15d
+
+# In[89]:
+
+get_ipython().magic('matplotlib inline')
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import math
+
+
+# In[90]:
+
+data = pd.read_csv('../data/2017-03-14-reimbursements.xz',
+                   parse_dates=[16],
+                   low_memory=False,
+                   dtype={'document_id': np.str,
+                          'congressperson_id': np.str,
+                          'congressperson_document': np.str,
+                          'term_id': np.str,
+                          'cnpj_cpf': np.str,
+                          'reimbursement_number': np.str,})
+
+
+# First step is subset the data of local transportation of all reimbursements
+
+# In[91]:
+
+taxi = data[(data['subquota_description'] == 'Taxi, toll and parking')]
+fuel = data[(data['subquota_description'] == 'Fuels and lubricants')]
+rent_car_and_boat = data[(data['subquota_description'] =='Automotive vehicle renting or watercraft charter')]

```

Comment:
Here I did some analysis again, for this reason I think is better don't link the others notebooks.